docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -122,6 +123,25 @@ components:
   # Reusable parameters
   # -------------------------
   parameters:
+    PageNumber:
+      name: "page[number]"
+      in: query
+      required: false
+      description: Which page of results to get
+      schema:
+        type: int
+        default: 1
+        minimum: 1
+    PageSize:
+      name: "page[size]"
+      in: query
+      required: false
+      description: How many items to get per page
+      schema:
+        type: int
+        default: 20
+        minimum: 1
+        maximum: 100
```

Comment:
Putting the pagination query params in a reusable place. They're currently only used by one endpoint but if we ever add another endpoint with page-based pagination we'll want to re-use the same query params. That said I don't know if/how you can customize the `default` and `maximum` values when reusing a parameter in OpenAPI. :man_shrugging: A problem for another day

Reply:
> That said I don't know if/how you can customize the `default` and `maximum` values when reusing a parameter in OpenAPI. 🤷‍♂️ A problem for another day

You can use `allOf` where one of the "sides" is a `$ref` to the common parts of the schema, and the rest are the parts that are different.